llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -557,11 +557,6 @@ static void initialize(TargetLibraryInfoImpl &TLI, const Triple &T,
     }
     break;
   case Triple::Linux:
-    // exp10, exp10f, exp10l is available on Linux (GLIBC) but are extremely
-    // buggy prior to glibc version 2.18. Until this version is widely deployed
-    // or we have a reasonable detection strategy, we cannot use exp10 reliably
-    // on Linux.
-    //
```

Comment:
I would leave some kind of historical note

Reply:
Oh okay, thanks again!

Reply:
Just update the "Until this version is widely deployed or we have a reasonable detection strategy, we cannot use exp10 reliably" part to something about how this is so old it doesn't matter

Reply:
"As this version is so old, we don't really need to worry about using exp10 until we find more good ways to detect it."

Does this look fine?

Reply:
The "until we find more good ways to detect it." reads off. Can drop it or rephrase

Reply:
Yes, I will drop this.